src/ninja.cc

```diff
@@ -827,6 +830,8 @@ const Tool* ChooseTool(const string& tool_name) {
       Tool::RUN_AFTER_LOAD, &NinjaMain::ToolCompilationDatabase },
     { "recompact",  "recompacts ninja-internal data structures",
       Tool::RUN_AFTER_LOAD, &NinjaMain::ToolRecompact },
+    { "cleandead",  "clean built files that are no more produced by the manifest",
```

Comment:
I'm not a native English speaker, but shouldn't this be "are not produced anymore" instead?

Reply:
Not a native speaker either... Cambridge says the « no more » is more formal than « not any more », and that « any more » is more English than « anymore »(one word) is more American English... so ... you decide what you prefer ;-)

Reply:
Okay, it's fine then.

Reply:
How about "that are no longer produced by the manifest"?